playbooks/adjust-cinder-backend.yml

```diff
@@ -47,6 +47,12 @@
       when: not cinder.enabled|default('False')|bool
       run_once: true
 
+    - fail: msg="Only one type of osds should be configured"
```

Comment:
I tried check-cinder-service.sh under sensu plugin dir. it didn't work very well.
the sensu check can make sure service is up, while it can promise `cinder type-creat` works.
I use `cinder list` instead